### ISSUE TITLE
[Bug]: desktop client fails to build since weeks, openSUSE Tumbleweed

### ISSUE THREAD

**Issue by Keks-Dose** [nextcloud/desktop#4690]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

There is a bug report on the bugzilla of openSUSE and it seems to turn out that the client fails to build. See here:

https://bugzilla.opensuse.org/show_bug.cgi?id=1201070

> Problem: nothing provides 'libQt5Gui.so.5(Qt_5.15.2_PRIVATE_API)(64bit)' needed by the to be installed nextcloud-desktop-3.5.1-1.1.x86_64

I'm a user and can't give more information, but I'd be glad if someone could have a look. I've been bitten by this myself, so this isn't a second hand report. 

### Steps to reproduce

After an update to a newer version of libQt5 than mentioned above nextcloud won't start. 

Used desktop client version is 3.5.1

### Expected behavior

Obviously... 

### Which files are affected by this bug

/usr/bin/nextcloud

### Operating system

Linux

### Which version of the operating system you are running.

openSUSE Tumbleweed

### Package

Distro package manager

### Nextcloud Server version

This is about the desktop software

### Nextcloud Desktop Client version

3.5.1

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell
Nonsense, I'm complaining about the desktop software, that fails to build!
```

**Comment by osm-frasch** [comment]
Can completely confirm the above

**Comment by arjanjnl** [comment]
It doesn't look like a bug in the code but in the rpm package and it seems to be solved:
https://build.opensuse.org/package/show/network/nextcloud-desktop

**Comment by sebix** [comment]
> It doesn't look like a bug in the code but in the rpm package and it seems to be solved: https://build.opensuse.org/package/show/network/nextcloud-desktop

Yeah, the bug reported here is a packaging issue and already being dealt with, but the underlying reason is explained in https://bugzilla.opensuse.org/show_bug.cgi?id=1201070#c3

**Comment by bnavigator** [comment]
> It doesn't look like a bug in the code

Well, there must be a bug in the code which was uncovered by `FORTIFY_SOURCE=3`.

**Comment by osm-frasch** [comment]
> It doesn't look like a bug in the code but in the rpm package and it seems to be solved: https://build.opensuse.org/package/show/network/nextcloud-desktop

No, unfortunately it is not yet solved. I get the following when I run zypper install nextcloud-client: 
`Problem: nothing provides 'libQt5Gui.so.5(Qt_5.15.2_PRIVATE_API)(64bit)' needed by the to be installed nextcloud-desktop-3.5.1-1.1.x86_64`

**Comment by arjanjnl** [comment]
The package was updated but it will be available in the next Tumbleweed snapshot. This is from the Opensuse factory mailinglist:

The package has been fixed and submitted into Factory, where it's
alrady accepted.

It will be part of Snapshot 20220701.

**Comment by claucambra** [comment]
Hi, since the issue with the package has been worked around I will close this issue, keeping #4697 open
